src/statusBar/index.ts

```diff
@@ -256,3 +256,50 @@ export function createStatusBarItem(
   }
   return statusBarItem;
 }
+
+// --- ESP-IDF Hints Status Bar Item ---
+let hintsStatusBarItem: StatusBarItem | undefined;
```

Comment:
Remove in favor of already existing `statusBarItems: { [key: string]: StatusBarItem }` dictionary.